utils/arrange-json.js

```diff
@@ -0,0 +1,44 @@
+const fs = require("fs");
+const icons = require("./eos-icons.json");
```

Comment:
Now I see that you import `eos-icons`, in this case we should programmatically generate the list below by grabbing all the categories, and then remove all duplications with a js `filter` or a `map` or a `reduce`

Reply:
Same as above.